servers.json

```diff
@@ -837,5 +837,19 @@
         "addresses": [
             "beastmc.net"
         ]
+    },
+    {
+        "id": "vanir",
+        "name": "Vanir Network",
+        "addresses": [
+            "vanir.cc"
+        ]
+    },
+    {
+        "id": "vanir",
```

Comment:
You do not need to add entries for each subdomain, Lunar Client services already check for subdomains. Please remove this entry.

Reply:
Please refer to [the address section](https://github.com/LunarClient/ServerMappings#ip-addresses) of the README for information on formatting for server addresses.

Reply:
fixed